tests/Console/ConfigurationResolverTest.php

```diff
@@ -961,6 +961,26 @@ public function provideResolveRulesCases()
                     '@A' => true,
                 ),
             ),
+            '@A - @C' => array(
+                array(
+                    'AB' => true,
+                    'AC' => 'a',
+                ),
+                array(
+                    '@A' => true,
+                    '@C' => false,
+                ),
+            ),
+            '@A - @D' => array(
```

Comment:
I agree with @keradus on `[ AC => false ]` being the most consistent way to turn AC off right now. That is considering we don't want to change the way rules that contain arrays are set.

But what if we set rules that contain arrays the same way we set regular rules? For example:
in `@A` rule `AC=['a' => true]`
in `@D` rule `AC=['b' => true]`

It would be more consistent and would allow to fine tune which rules are turned on and off. For example:
in `@A` rule `AC=['a' => true]`
in `@D` rule `AC=['b' => true, 'a' => false]`
This would work as @SpacePossum proposed and be fairly explicit.